### PR TITLE
fix stylus includes

### PR DESCRIPTION
**Pull request by studentIvan**
1. fix stylus extension (from .stylus to .styl)
2. fix default stylus includes in app.styl (account/login/login contains nothing and it does produce some err)

### CODE REVIEW

File: app/templates/client/app/app(stylus).styl [hunk 1]
```diff
@@ -45,7 +45,6 @@
 <% } %>
 // Component styles are injected through grunt
 // injector
-@import "account/login/login"
```
Comment (Awk34) [inline app/templates/client/app/app(stylus).styl]: ?

File: app/generator.js [hunk 2]
```diff
@@ -144,7 +144,9 @@ export default class Generator extends Base {
 
             this.scriptExt = answers.transpiler === 'ts' ? 'ts' : 'js';
             this.templateExt = answers.markup;
-            this.styleExt = answers.stylesheet === 'sass' ? 'scss' : answers.stylesheet;
+
+            var styleExt = {sass: 'scss', stylus: 'styl'}[answers.stylesheet];
+            this.styleExt = styleExt ? styleExt : answers.stylesheet;
```
Comment (Awk34) [inline app/generator.js]: This should be shortened to something like

```js
this.styleExt = answers.stylesheet === 'sass' ? 'scss' :
  answers.stylesheet === 'stylus' ? 'styl' :
  answers.stylesheet;

File: app/templates/client/app/app(stylus).styl (hunk 1 above)
Comment (studentIvan) [inline app/templates/client/app/app(stylus).styl]: this file doesn't exists

File: app/generator.js (hunk 2 above)
Comment (studentIvan) [inline app/generator.js]: no problem, any way

File: app/templates/client/app/app(stylus).styl (hunk 1 above)
Comment (Awk34) [inline app/templates/client/app/app(stylus).styl]: Oh you're right, curious, it *used* to exist. Anyway, if you're going to remove this, it should be in a separate commit, and you should do the same change for the SCSS and LESS files.

**Comment by Awk34** [comment]
Thanks. I've pulled out the app.styl change and merged the rest.
